src/erasure-code/shec/CMakeLists.txt

```diff
@@ -1,49 +1,36 @@
 #shec plugin
 
-set(shec_srcs
+include_directories(.)
+include_directories(${CMAKE_SOURCE_DIR}/src/erasure-code/jerasure/jerasure/include)
+include_directories(${CMAKE_SOURCE_DIR}/src/erasure-code/jerasure/gf-complete/include)
+include_directories(${CMAKE_SOURCE_DIR}/src/erasure-code/jerasure)
```

Comment:
@cbodley include_directories only applies to targets in a CMakeLists.txt, so I can either leave them as be or put them in erasure-code/CMakeLists.txt